### PR TITLE
Feature/directories

### PR DESCRIPTION
**Pull request by diegoaad**
Hi Phil, in this branch I'm fixing issue #88. I have updated the Directory page to add the columns according to the mockup. This included changing the db to add mobile and home phone columns.
Also I added a new option on user's setting allowing them to hide/show their profile in the public directory. This also required changed in the db.
Besides I have created a new listing called "Organizers" that should list only people with roles in the circle and working group organizers.

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class SessionsController < ApplicationController
 
   def new
     if current_user.present?
+      current_user.update_attribute(:last_login, Time.now)
```
Comment (phil-monroe) [inline app/controllers/sessions_controller.rb]: For the future, use the Rails defined method [`touch`](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-touch)

File: app/models/user.rb [hunk 2]
```diff
@@ -35,11 +35,15 @@ def login_token
...
     "#{first_name} #{last_name}"
   end
-
+  
   def email
     identity.try :email
   end
-
+  
+  def active_since
```
Comment (phil-monroe) [inline app/models/user.rb]: nit: use [`alias_method`](http://ruby-doc.org/core-2.3.0/Module.html#method-i-alias_method) over manually defining a method

File: app/views/circle/members/index.slim [hunk 3]
```diff
@@ -1,22 +1,48 @@
...
+	      td
+	      	=render partial: 'user/user_icon', locals: {user: user}
+	      	=user.name      
+	      td
+	        - user.working_groups.for_circle(current_circle).each do |wg|
+	          a href= circle_working_group_path(current_circle,wg) 
+	            =wg.name
+	      td= user.location.address
+	      td
+	        - if user.mobile_phone != nil
```
Comment (phil-monroe) [inline app/views/circle/members/index.slim]: nit: prefer [`.present?`](http://api.rubyonrails.org/classes/Object.html#method-i-present-3F) or [`.blank?`](http://api.rubyonrails.org/classes/FalseClass.html#method-i-blank-3F) over nil checks. It tends to be more readable.

File: app/models/user.rb (hunk 2 above)
Comment (phil-monroe) [inline app/models/user.rb]: ammendment: I didn't know this, but you cannot use `alias_method` with ActiveRecord attributes... However, Rails has [`alias_attribute`](http://api.rubyonrails.org/classes/Module.html#method-i-alias_attribute) which accomplishes the same :smile:

File: app/views/user/account/edit.slim [hunk 4]
```diff
@@ -41,7 +52,13 @@ main
         = f.label :primary_circle_id
         = f.collection_select :primary_circle_id, @form.primary_circle_options, :id, :name
         = errors.message_for(:primary_circle_id)
-
+        
+    .field-row
+      = f.check_box :public_profile
+      = " "+User.human_attribute_name("public_profile")
```
Comment (phil-monroe) [inline app/views/user/account/edit.slim]: Don't use a Ruby `String` to add padding. Ruby is for functionality, CSS is for styling.

**Comment by phil-monroe** [comment]
Hey @diegoaad, for the most part everything is great! Just had a couple nit-picks that I took care of for you. Just take a look at the outdated diffs. Cheers!

**Comment by diegoaad** [comment]
Hi @phil-monroe I reviewed the diffs, will keep these in mind in the future. Thanks a lot for the review and the changes.

**Pull request merged**